support_bundle/get_support_bundle.sh

```diff
@@ -141,8 +141,17 @@ main() {
 
     # If API key is supplied, collect streamSnap, Index settings, and fastPath settings
     if [[ ! -z ${API_KEY} ]]; then
-        API_URL=$(kubectl ${KUBE_OPTS} get cm sysdigcloud-config -o yaml | grep -i api.url: | head -1 | awk '{print$2}')
-
+        #Check if we are on version 6
+        VERSION_CHECK=$(kubectl ${KUBE_OPTS} get cm | grep 'sysdigcloud-api-config' | wc -l | sed 's/       //g') || true
```

Comment:
@patrickhargett not sure why we have the `sed` command here.

Suggest we simplify this using `-c` on grep to give us the word count on the output.
`VERSION_CHECK=$(kubectl ${KUBE_OPTS} get cm | grep -c 'sysdigcloud-api-config' || true`